### PR TITLE
Adds lapotron dust batch recipe and enderium skip

### PR DESCRIPTION
**Pull request by BlueHero233**
Adds a way to skip enderium base and a batch recipe for lapotron dust

### CODE REVIEW

**Comment by chochem** [comment]
why a batch recipe? that is done on the level of machines

**Comment by BlueHero233** [comment]
> why a batch recipe? that is done on the level of machines

went right over my head, let me fix that

**Comment by BlueHero233** [comment]
> I don't think a Lapotron Dust skip is a good idea. I'm fine with Enderium though (but, why MV?)

Just a placeholder voltage, Enderium is LV though, maybe jot that up to HV and lower the time?

**Comment by chochem** [comment]
I also changed the label on both because direct recipes are not an _enhancement_, but just a way to make the game easier. Hence balance.

File: src/main/java/com/dreammaster/gthandler/recipes/MixerRecipes.java [hunk 1]
```diff
@@ -490,6 +490,20 @@ public void run() {
...
+                        GTOreDictUnificator.get(OrePrefixes.dust, Materials.Silver, 32L),
+                        GTOreDictUnificator.get(OrePrefixes.dust, Materials.Platinum, 32L),
+                        GTOreDictUnificator.get(OrePrefixes.dust, Materials.Thaumium, 64L),
+                        GTOreDictUnificator.get(OrePrefixes.dust, Materials.EnderPearl, 64L),
+                        GTUtility.getIntegratedCircuit(12))
+                .itemOutputs(GTOreDictUnificator.get(OrePrefixes.dust, Materials.Enderium, 64L),
+                        GTOreDictUnificator.get(OrePrefixes.dust, Materials.Enderium, 64L),
+                        GTOreDictUnificator.get(OrePrefixes.dust, Materials.Enderium, 64L),
+                        GTOreDictUnificator.get(OrePrefixes.dust, Materials.Enderium, 64L)).duration(16 * MINUTES)
+                .eut(TierEU.RECIPE_MV).noOptimize().addTo(mixerRecipes);
```
Comment (Dream-Master) [inline src/main/java/com/dreammaster/gthandler/recipes/MixerRecipes.java:505]: no sorry but not mv. To skip enderium bas it must be a few tier higher

**Review by Dream-Master — CHANGES_REQUESTED** [review]

File: src/main/java/com/dreammaster/gthandler/recipes/MixerRecipes.java (hunk 1 above)
Comment (Dream-Master) [reply src/main/java/com/dreammaster/gthandler/recipes/MixerRecipes.java]: why github show this is outdated? its not

Comment (BlueHero233) [reply src/main/java/com/dreammaster/gthandler/recipes/MixerRecipes.java]: Im in the middle of fixing it right now, probably gonna close this PR as the enderium recipe will be added in the ABS and the lapotron recipe removed

**Pull request closed without merging**